### PR TITLE
DEV-36739 Fix default namespace resolution

### PR DESCRIPTION
**Pull request by willynganga**

### CODE REVIEW

File: src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java [hunk 1]
```diff
@@ -212,6 +220,25 @@ private static Document buildDocument(String xmlContent)
     }
   }
 
+  private static Document buildDocumentWithoutNamespaceAwareness(String xmlContent)
+      throws ParserConfigurationException, IOException, SAXException {
+    DocumentBuilderFactory documentBuilderFactory =
```
[the remark this answers is not in the document]
Comment (willynganga) [reply src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java:225]: Correct me if I'm wrong, but I think you are opposed to having a boolean as an argument, right?

[the remark this answers is not in the document]
Comment (willynganga) [reply src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java:230]: Understood, I have updated it.

File: src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java [hunk 2]
```diff
@@ -187,15 +192,23 @@ private static String getDocumentXml(Document document) throws IOException {
 
   private static Document buildDocument(String xmlContent)
```
Comment (Thilo-Acrolinx) [inline src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java:193]: for consistency, we should rename this method to "buildDocumentWithNamespaceAwareness" I think?

File: src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java [hunk 3]
```diff
@@ -56,6 +56,11 @@ public static IntRange findOffsetForNodeInXmlStringByXpath(String xmlContent, St
       NodeList nodeList =
           (NodeList) xPath.compile(xpath).evaluate(document, XPathConstants.NODESET);
 
+      if (nodeList.getLength() == 0) { // load document without context awareness
```
Comment (Thilo-Acrolinx) [inline src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java:59]: I think the method name in the next line already conveys this information, so we could omit the comment

**Pull request merged**